docs/In-browserExecution.md

```diff
@@ -0,0 +1,65 @@
+# In-browser Execution for Compiler Explorer
+
+## What is it?
+It is an enhancement feature of Compiler Explorer (CE) by adapting it to support in-browser compilers. The primary objectives include restructuring the backend by removing the original CE server and integrating WebAssembly (Wasm) code for compiler execution. Additionally, the project aims to add a new feature of running different compiler passes so that users can better understand the compiler process. Furthermore, the project implements another functionality to load different compilers.
+
+## Build process
+
+### Installing Prerequisites
```

Comment:
Why are we installing node and npm using pip (which is a python dependency manager)? Why not just do brew install node for macOS

Reply:
I changed it!

Reply:
And npm gets installed with node so you don't have to install it separately.

Reply:
Can you also mention that brew is only for macOS. For windows or some other OS maybe add a link from the internet that explain how to do the installation